src/service/localization.js

```diff
@@ -94,6 +95,11 @@ export class LocalizationService {
   constructor(win) {
     const rootEl = win.document.documentElement;
 
+    /**
+     * @private {?string}
+     */
+    this.viewerLanguageCode_ = parseQueryString(win.location.hash)['lang'];
```

Comment:
`ampdoc.getParam('lang')` or `viewerService.getParam('lang')` should give you that. No idea if that's working around some edge cases or anything, but it's what's mostly used to retrieve URL parameters.

Reply:
Could we make it explicit that either this fragment mechanism, or this language service will only work properly with render-blocking components like `amp-story`?

Reply:
@gmajoulet updated to use `viewer.getParam('lang')`; PTAL
@alanorozco I don't think that necessarily needs to be the case?  Any UIs that are lazy loaded could also use this.  But, this is more an issue with the `LocalizationService` itself, than with the viewer language override, right?

Reply:
How are values for `viewer.getParam` read?

If it's a fragment, it will definitely be lazy UI. There's rarely any lazy UI outside stories, and it would cause content shift in most cases unless it render-blocks. I would like this to be explicit, even just with a comment if we can't do it by naming.

Reply:
@alanorozco but, there's no way to use the service at all except for lazily-rendered UIs, right?  This doesn't make it any worse or better.